hudi-client/src/main/java/org/apache/hudi/HoodieWriteClient.java

```diff
@@ -824,14 +843,14 @@ private String startInstant() {
           "Found commits after time :" + lastCommit + ", please rollback greater commits first");
     }
 
-    List<String> inflights =
-        inflightCommitTimeline.getInstants().map(HoodieInstant::getTimestamp).collect(Collectors.toList());
+    List<String> inflights = inflightAndRequestedCommitTimeline.getInstants().map(HoodieInstant::getTimestamp)
```

Comment:
Does rollback need inflight and requested both ? The restore command probably needs both but rollback should only consider inflight ?

Reply:
So, that's true for other types of actions but for compaction we need to also delete the extra log file created (I'm guessing this part of the code works as expected already and isn't changed as part of this diff, correct ? )

Reply:
Yes, it should work fine. For restoring (reverting completed) compactions, we would still delete log files as part of first rolling-back the inflight state of the compaction before we try to rollback requested state.